### PR TITLE
curvefs_python: add build options for python2/3

### PR DESCRIPTION
**Pull request by wu-hanqing**
### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: mk-deb.sh [hunk 1]
```diff
@@ -54,6 +54,19 @@ fi
 
 curve_version=${tag_version}+${commit_id}${debug}
 
+PYTHON_BIN=python2
+PYTHON_CONF_OPTS=python2
+
+if [ $# -ge 2 ]; then
+    if [ "$2" = "python2" ] || [ "$2" = "python3" ]; then
+        PYTHON_BIN=$2
+        PYTHON_CONF_OPTS=$2
```
Comment (bai-charisu) [inline mk-deb.sh:63]: mk-deb现在必须指定两个参数？

File: mk-deb.sh [hunk 2]
```diff
@@ -114,7 +127,7 @@ then
     echo "build phase1 failed"
     exit
 fi
-bash ./curvefs_python/configure.sh
+bash ./curvefs_python/configure.sh ${PYTHON_CONF_OPTS}
```
Comment (bai-charisu) [inline mk-deb.sh:130]: PYTHON_CONF_OPTS在哪里定义的？

File: mk-deb.sh (hunk 1 above)
Comment (wu-hanqing) [reply mk-deb.sh:63]: 如果默认用python2，直接mk-deb/mk-tar就行了。如果要python3，就mk-deb.sh release python3

File: mk-deb.sh (hunk 2 above)
Comment (wu-hanqing) [reply mk-deb.sh:130]: 57 58行定义的。

**Review by bai-charisu — APPROVED** [review]

File: mk-deb.sh [hunk 3]
```diff
@@ -54,6 +54,19 @@ fi
...
 
+PYTHON_BIN=python2
+PYTHON_CONF_OPTS=python2
+
+if [ $# -ge 2 ]; then
+    if [ "$2" = "python2" ] || [ "$2" = "python3" ]; then
+        PYTHON_BIN=$2
+        PYTHON_CONF_OPTS=$2
+    else
+        echo "Usage: ./mk-deb.sh [debug|release] [python2|python3]"
```
Comment (xu-chaojie) [inline mk-deb.sh:65]: 我觉得，是否不要通过加参数的方式，直接把两个包都生成好了

**Review by xu-chaojie — APPROVED** [review]

**Pull request merged**